### PR TITLE
New tests organization

### PR DESCRIPTION
**Pull request by rjourde**

### CODE REVIEW

File: models/user_helpers_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		if id == got.Id {
+			return nil
+		}
+	}
+
+	return errors.New("user not found")
+
+}
+
+func createNonSavedUser(email, username, name, alias string, isAdmin bool) User {
```
Comment (santiaago) [inline models/user_helpers_test.go]: could we add the field names on this structure?
`Description: `, `Name`, etc

**Comment by santiaago** [comment]
did you run all tests? :)

File: models/user_helpers_test.go (hunk 1 above)
Comment (rjourde) [inline models/user_helpers_test.go]: which structure are you talking about ?

**Comment by rjourde** [comment]
All tests are OK!

File: models/user_helpers_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		[]int64{},
+		[]int64{},
+		[]int64{},
+		[]int64{},
+		[]int64{},
+		0,
+		[]ScoreOfTournament{},
+		[]int64{},
+		time.Now(),
+	}
```
Comment (santiaago) [inline models/user_helpers_test.go]: this one

Comment (rjourde) [inline models/user_helpers_test.go]: ok. I do it.

**Comment by rjourde** [comment]
Done!

**Comment by santiaago** [comment]
:+1:

**Pull request merged**